### ISSUE TITLE
Running services inside containers

### ISSUE THREAD

**Issue by smoll** [apparatus/fuge#171]
Perhaps I'm misunderstanding how this is supposed to work, but I assumed that `fuge` would allow me to start a service using the Dockerfile defined in an included compose file.

Looking at the examples, though, it looks like all it does is read the config from a compose file, and translate it into running the process locally (not through docker): https://github.com/apparatus/fuge-examples/tree/master/using-docker-compose

Is this use case not supported by fuge? Basically, I want to keep using docker-compose, just using fuge to prevent shell hell.

**Comment by pelger** [comment]
If you look at the example you will see that some of the processes are defined in the fuge.yml file. The docker-compose.yml file defines a mongo container and the auditservice. Because the audit service has the `build` specified, when fuge loads this configuration it follows the build path for the audit file and attempts to load a Dockerfile at that location. It parses this file and extracts the CMD entry. It will use the CMD entry to run the process rather than starting a container. 

If you edit the CMD entry in the auditservice Dockerfile you will see that the change is reflected in the fuge shell on restart.

Hope that makes sense!

**Comment by smoll** [comment]
Thanks for the reply @pelger, but based on

> It will use the CMD entry to run the process rather than starting a container.

it sounds like it will simply run `node index.js` in the host machine, which is not what I want. If my application requires, for example, `node-postgres` bindings, users would have to install and configure `libpq` for their host machine (or some other library that has a slightly different install process depending on if you're on Mac, Windows, Linux, etc.), which is something I want to avoid.

Am I correct that my preferred usage is not actually supported by fuge?

**Comment by pelger** [comment]
Yes thats correct. Generally I try to avoid modules that need native bindings on the machine, however in the case you outline these would be required.
